### ISSUE TITLE
Vengeance Other overlay on others

### ISSUE THREAD

**Issue by ZuenKramer** [runelite/runelite#12175]
Was wondering if we could get a "Vengeance Others" overlay shown on one of your party members when you're raiding (mainly useful in Theatre of Blood) so you can see who is venge'd and who isn't. I've seen in a video once (can't find it anymore) that it was indicated by a white skull from the Vengeance Others spell on the person itself. Would be a very nice QoL for raiding.

**Comment by Hydrox6** [comment]
If implemented, this would have to be very carefully done so, as having it track anywhere near PvP could come under `Offers additional information about other players, with the purpose of scouting PvP targets` in [Jagex's Statement](https://secure.runescape.com/m=news/another-message-about-unofficial-clients?oldschool=1). Having it locked to both raids and their lobbies if not in a PvP world should be fine though.

**Comment by ZuenKramer** [comment]
Yes I agree it could have a negative impact on PvP and that's not what I am looking for of course, I don't know if they can trigger the plug-in to just work in certain parts of the game like CoX (don't think it's necessary there anyway) or ToB for example.
